src/lib.rs

```diff
@@ -25,18 +26,23 @@
 //! # #[allow(unused_variables)]
 //! fn main() {
 //!     let mut book =  MDBook::new("my-book")        // Path to root
-//!                         .with_source("src")       // Path from root to source directory
-//!                         .with_destination("book") // Path from root to output directory
-//!                         .read_config()            // Parse book.toml configuration file
-//!                         .expect("I don't handle configuration file errors, but you should!");
+//! .with_source("src")       // Path from root to
```

Comment:
rustfmt seems to have destroyed the code snippets in the doc comments and this results in test failures.
This is certainly a bug in rustfmt.

Reply:
Yeah, that seems to be the case.

Reply:
So, this means we can't use rustfmt currently?

Reply:
> So, this means we can't use rustfmt currently?

Not really. We can pull only some of the changes that improve the code readability. Also it might be a good moment to submit bug report to rustfmt

Reply:
Hey @budziq, I closed this pull request and then opened a new pull request fixing the problem. I did write an issue on rustfmt and found the solution to this.